### ISSUE TITLE
Remove "Maximum upload size" from System Settings

### ISSUE THREAD

**Issue by pgorod** [salesagility/SuiteCRM#8393]
## Issue 

In `Admin` / `System Settings` we have a setting for `Maximum upload size`. 

This setting apparently only affects the way SuiteCRM tests file sizes before attempting upload; but we don't change the PHP setting from this value, `upload_max_filesize`, because that's not possible (`ini_set` doesn't work for this, [by design](https://www.php.net/manual/en/ini.list.php), since PHP 5.3).

Then there are the settings in php.ini which are the ones that actually take effect and cause PHP to FATAL out when trying a larger upload.

So this setting is ineffective and misleading. For example: our default is `30000000`. If the user has a larger value effective in `php.ini`, something quite common nowadays, some upload code paths (but not _all_ uploads in SuiteCRM) will fail with an error message saying that the upload limit is the problem, but it won't be obvious that it's the limit in `Admin` / `System Settings`, people will just think it's the `php.ini` value. But changing _that_ won't solve the error message.

On the other hand, if people have a higher value in `php.ini` than in System Settings, our error messages won't kick in, and they will get an uglier FATAL instead.

We also have a bunch of places in the code where we mess with this value, give it 8MB, 60MB, not really changing anything except the error threshold.

## Proposed solution

If the option isn't changing anything for real, it shouldn't exist.

- remove field from `Admin` / `System Settings`
- set `$sugar_config['upload_maxsize']` to whatever `ini_get('upload_max_filesize')` returns
- remove any instances of changing `$sugar_config['upload_maxsize']` in our code

... unless I'm missing something. I did review the code for everything I could find related to these values, but it is possible something escaped me.

**Comment by Dillon-Brown** [comment]
Agreed, although I think the solution will need to be a little more complicated since ```ini_get('upload_max_filesize')``` won't return the actual value we need as it could be in [various different formats](https://www.php.net/manual/en/faq.using.php#faq.using.shorthandbytes) or even set to 0 for no limits. I'm guessing we'd need to parse the ```upload_max_filesize``` along with the ```post_max_size``` before then calculating a max size.

I'll see if I can throw up a potential solution.
